### PR TITLE
fix: ball lose

### LINKED ISSUE #38337: Anchor 锚点的球没有了
### Reproduction link

[https://ant.design/components/anchor-cn/#components-anchor-demo-basic](https://ant.design/components/anchor-cn/#components-anchor-demo-basic)

### Steps to reproduce

看demo

### What is expected?

锚点前面的球存在

### What is actually happening?

锚点前面的球没有了

| Environment | Info |
| --- | --- |
| antd | 4.24.0 |
| React | antd版本 |
| System | macOS 13.0 |
| Browser | Edg/107.0.1418.24 |

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

- close #38337 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix: ball lose |
| 🇨🇳 Chinese | fix: ball lose |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/anchor/style/index.less [hunk 1]
```diff
@@ -44,7 +44,7 @@
       transform: translateX(-50%);
       transition: top 0.3s ease-in-out;
 
-      &.@{ant-prefix}-ink-ball-visible {
+      &.ink-ball-visible {
```
Comment (afc163) [inline components/anchor/style/index.less:47]: 上面不用动，这里改成

```suggestion
      &.@{ant-prefix}-anchor-ink-ball-visible {
```

File: components/anchor/Anchor.tsx [hunk 2]
```diff
@@ -229,7 +229,7 @@ const AnchorContent: React.FC<InternalAnchorProps> = props => {
 
   const inkClass = classNames(
     {
-      [`${prefixCls}-ink-ball-visible`]: activeLink,
+      [`ink-ball-visible`]: activeLink,
```
Comment (afc163) [inline components/anchor/Anchor.tsx:232]: 这里不用改

Comment (li-jia-nan) [reply components/anchor/Anchor.tsx]: 我改回去了

**Review by afc163 — APPROVED** [review]

**Comment by li-jia-nan** [comment]
@afc163 大佬这个算严重的 bug 吗？要不要发个 patch version ？

**Comment by afc163** [comment]
我觉得还好，不影响功能。

**Pull request merged**